dyninstAPI/src/inst-aarch64.C

```diff
@@ -1389,37 +1422,257 @@ bool EmitterAARCH64::emitCallInstruction(codeGen &gen, func_instance *callee, bo
 
 void EmitterAARCH64::emitLoadShared(opCode op, Register dest, const image_variable *var, bool is_local, int size,
                                     codeGen &gen, Address offset) {
-    assert(0); //Not implemented
+    //assert(0); //Not implemented
+    //return;
+    
+    // create or retrieve jump slot
+    Address addr;
+    int stackSize = 0;
+
+    if(var == NULL) {
```

Comment:
In C++11, `nullptr` is prefered over NULL. The common idiom of `if(var)` would also be fine.

Reply:
That should be `if(!var)` in this case.